samples/bluetooth/throughput/README.rst

```diff
@@ -10,12 +10,12 @@ Bluetooth: Throughput
 The Bluetooth Throughput sample uses the :ref:`throughput_readme` to measure *Bluetooth* Low Energy throughput performance.
 You can use it to determine the maximum throughput, or to experiment with different connection parameters and check their influence on the throughput.
 
-
 Overview
 ********
 
 The sample transmits data between two boards, the *tester* and the *peer*, and measures the throughput performance.
-To do so, it uses the :ref:`throughput_readme`.
+To do so, it uses the :ref:`throughput_readme`. You can run the tests by connecting to the board through the serial port and sending shell commands.
+Zephyr's :ref:`zephyr:shell_api` module is used to handle the commands
```

Comment:
Proposing:
It uses the ...
To run the tests, connect to the board using the serial port and send shell commands.

Reply:
Done